vspace/src/main/webapp/WEB-INF/views/staff/exhibit/config.jsp

```diff
@@ -0,0 +1,28 @@
+<%@ page pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="t"%>
+<%@ taglib prefix="sec"
+	uri="http://www.springframework.org/security/tags"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+
+<h1>Exhibition Configuration</h1>
+
+<div style="padding-bottom: 20px;">
+	Select your exhibition and default space here
+	<c:url value="/staff/exhibit/config" var="postUrl" />
+	<form:form method="POST"
+		action="${postUrl}?${_csrf.parameterName}=${_csrf.token}"
+		modelAttribute="exhibit,spaces">
```

Comment:
that looks very odd, why are you doing that? typically there is just one model attribute (I don't think it works with more than one), and you don't even use model attributes in the controller.